engine/build.gradle

```diff
@@ -150,10 +150,12 @@ dependencies {
     api group: 'org.terasology', name: 'gestalt-module', version: '5.1.5'
     api group: 'org.terasology', name: 'gestalt-util', version: '5.1.5'
     api group: 'org.terasology', name: 'gestalt-asset-core', version: '5.1.5'
-    api group: 'org.terasology', name: 'TeraMath', version: '1.4.0'
+    api group: 'org.terasology', name: 'TeraMath', version: '1.5.1-SNAPSHOT'
```

Comment:
Why do we depend on a snapshot version here? Can we do a proper release for TeraMath, please?

Reply:
No particular reason, same situation as TeraBullet I think, just been a low activity area + there's a looming replacement on the horizon (JOML)

Has a similar build job in the old Jenkins which probably works, but hasn't been touched for nearly two years: http://jenkins.terasology.org/view/Ext/job/TeraMath/

It _looks_ like it is configured to handle either snapshots or releases from the `develop` branch, I would guess it will figure it out from the version property, wherever it is. Just can't guarantee it without trying it :-) Would need to be triggered manually like TeraBullet (old GitHub trigger is broken)

Reply:
There's only been [one commit](https://github.com/MovingBlocks/TeraMath/pull/64) since the TeraMath's last release. My hunch is that it's not necessary for the success of this PR, so I've rolled the dependency back to the last TeraMath release, 1.5.0.